packages/collector/test/tracing/database/mongodb/app.js

```diff
@@ -6,7 +6,7 @@
 'use strict';
 
 require('./mockVersion');
-const isLatest = process.env.MONGODB_VERSION === 'latest';
+const isLatest = process.env.MONGODB_VERSION === 'latest' || process.env.MONGODB_VERSION === 'v5';
```

Comment:
TBD: Wondering if we should rewrite this to `isLegacy` because v5 is not latest anymore.

Reply:
Can we consider this as modern versions? I changed the name to isModernVersion, please share your thoughts on this name.

Reply:
The name is better. But I think this is more about the majority of versions who are using the new code. There is only one legacy. But we can keep it as it is 👍